pytensor/tensor/variable.py

```diff
@@ -816,7 +816,9 @@ def compress(self, a, axis=None):
         return at.extra_ops.compress(self, a, axis=axis)
 
     def set(self, y, **kwargs):
-        """Set values to y, where y is the output of an index operation.
+        """Return a tensor whose indexed values are set to y.
```

Comment:
```suggestion
        """Return a copy of the calling tensor, where the indexed values are set to y.
```

Reply:
I meant self. I’m not sure how to call it

Reply:
But, it's not self, because this is not a `__getitem__`. self is the subtensor operation `x[idx]`.